src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -146,7 +146,33 @@ public IList<ITelemetryInitializer> TelemetryInitializers
         {
             get { return this.telemetryInitializers; }
         }
-        
+
+        /// <summary>
+        /// Gets or sets the first telemetry processor in the chain of processors.
+        /// </summary>
+        public TelemetryProcessorChain TelemetryProcessorChain
```

Comment:
the property should be just named TelemetryProcessors, "chainness" of it is captured in the type and the propertyname doesn't really need to highlight that.
this will also break the TelemetryConfigurationFactory logic above, which needs to be fixed.